### PR TITLE
228 remove location columns on organizations table and add zipcode to locations table

### PR DESCRIPTION
**Pull request by yuricarvalhop**
# 🔗 Issue
https://github.com/rubyforgood/pet-rescue/issues/228

# ✍️ Description
Removing following fields from Organization:
- City
- Country
- Zipcode

Adding following field to Location
- Zipcode

# 📷 Screenshots/Demos

### CODE REVIEW

File: app/views/organizations/home/index.html.erb [hunk 1]
```diff
@@ -3,7 +3,8 @@
     <div class="row d-flex align-items-center">
       <div class="col-xxl-5  col-xl-6 col-lg-6 col-12">
         <div>
-          <h1 class="display-2 fw-bold mb-3">Rescue pets in <%= Current.organization.city %> at <u class="text-warning"><span class="text-primary"> <%= Current.organization.name %></span></u></h1>
+          <%# Where should I get this city information from? Current.user is nil (page can be accesses without login) and Current.organization doesnt have location yet (should I link it on this PR?) %>
```
Comment (yuricarvalhop) [inline app/views/organizations/home/index.html.erb:6]: How should I proceed here?

Comment (yuricarvalhop) [reply app/views/organizations/home/index.html.erb:6]: Just saw [213](https://github.com/rubyforgood/pet-rescue/issues/213), so maybe re-introduce this information when that gets done (?)

Comment (kasugaijin) [reply app/views/organizations/home/index.html.erb:6]: Yes for now you can comment that out!

Comment (yuricarvalhop) [reply app/views/organizations/home/index.html.erb]: done!

**Review by kasugaijin — APPROVED** [review]
looks good to me!

**Pull request merged**